helm_deploy/hmpps-resettlement-passport-ui/files/grafana-dashboards/hmpps-resettlement-passport-dashboard.json

```diff
@@ -710,20 +829,96 @@
         "skipUrlSync": false,
         "sort": 0,
         "type": "query"
+      },
+      {
+        "current": {
+          "selected": true,
+          "text": "Bedford (HMP & YOI)",
+          "value": "Bedford (HMP & YOI)"
+        },
+        "datasource": {
+          "type": "prometheus",
+          "uid": "P5DCFC7561CCDE821"
+        },
+        "definition": "label_values(total_prisoners_count,prison)",
+        "hide": 0,
+        "includeAll": false,
+        "label": "Prison",
+        "multi": false,
+        "name": "prison",
+        "options": [],
+        "query": {
+          "query": "label_values(total_prisoners_count,prison)",
+          "refId": "StandardVariableQuery"
+        },
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 0,
+        "type": "query"
+      },
+      {
+        "current": {
+          "isNone": true,
+          "selected": false,
+          "text": "None",
+          "value": ""
+        },
+        "datasource": {
+          "type": "prometheus",
+          "uid": "P5DCFC7561CCDE821"
+        },
+        "definition": "label_values(total_prisoners_count)",
+        "hide": 0,
+        "includeAll": false,
+        "label": "All Prisoners",
+        "multi": false,
+        "name": "allPrisoners",
+        "options": [],
+        "query": {
+          "query": "label_values(total_prisoners_count)",
+          "refId": "StandardVariableQuery"
+        },
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 0,
+        "type": "query"
       }
     ]
   },
   "time": {
-    "from": "now-6h",
+    "from": "now-24h",
     "to": "now"
   },
   "timepicker": {
-    "refresh_intervals": ["5s", "10s", "30s", "1m", "5m", "15m", "30m", "1h", "2h", "1d"],
-    "time_options": ["5m", "15m", "1h", "6h", "12h", "24h", "2d", "7d", "30d"]
+    "refresh_intervals": [
+      "5s",
+      "10s",
+      "30s",
+      "1m",
+      "5m",
+      "15m",
+      "30m",
+      "1h",
+      "2h",
+      "1d"
+    ],
+    "time_options": [
+      "5m",
+      "15m",
+      "1h",
+      "6h",
+      "12h",
+      "24h",
+      "2d",
+      "7d",
+      "30d"
+    ]
   },
   "timezone": "browser",
   "title": "HMPPS Resettlement Passport",
   "uid": "fc317542-6db7-11ee-b962-0242ac120002",
-  "version": 6,
+  "version": 9,
```

Comment:
one day this versioning will make sense to me 😂

Reply:
kept as it was before to version 6